pkg/status/route.go

```diff
@@ -0,0 +1,299 @@
+// Copyright (c) 2021 Tigera, Inc. All rights reserved.
```

Comment:
For clarity, it might be worth organizing all of the populators into a sub-package. 

e.g.,

```
pkg/status
pkg/status/populators
```

Reply:
Good point.